web/controllers/admin/user_controller.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Pairmotron.AdminUserController do
+  use Pairmotron.Web, :controller
+
+  alias Pairmotron.User
+
+  @spec index(Plug.Conn.t, map()) :: Plug.Conn.t
+  def index(conn, _params) do
+    users = Repo.all(User)
+    render(conn, "index.html", users: users)
+  end
+
+  @spec new(Plug.Conn.t, map()) :: Plug.Conn.t
+  def new(conn, _params) do
+    changeset = User.changeset(%User{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  @spec create(Plug.Conn.t, map()) :: Plug.Conn.t
+  def create(conn, %{"user" => user_params}) do
+    changeset = User.changeset(%User{}, user_params)
```

Comment:
I think you should use `registration_changeset` so that the password validation happens.